Sources/SpotifyWebAPI/URLExtensions/URLSessionWrappers.swift

```diff
@@ -38,7 +38,7 @@ extension URLSession {
     /// This property exists so that it can be replaced with a different
     /// networking client during testing. Other than in the test targets, it
     /// will not be modified.
-    static var _defaultNetworkAdaptor: (
+    public static var _defaultNetworkAdaptor: (
```

Comment:
I don't want this property to be public. It's only meant to be changed during the tests. I am opposed to making mutable global symbols public.

Reply:
Fixed